x-pack/dockerlogbeat/pipelinemanager/pipelineManager.go

```diff
@@ -89,20 +106,82 @@ func (pm *PipelineManager) CreateClientWithConfig(containerConfig ContainerOutpu
 
 	reader, err := pipereader.NewReaderFromPath(file)
 	if err != nil {
-		return nil, errors.Wrap(err, "")
+		return nil, errors.Wrap(err, "error creating reader for docker log stream")
+	}
+
+	// Why is this empty by default? What should be here? Who knows!
+	if info.LogPath == "" {
+		info.LogPath = filepath.Join(pm.logDirectory, info.ContainerID)
+	}
+	err = os.MkdirAll(filepath.Dir(info.LogPath), 0755)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating directory for local logs")
+	}
+
+	localLog, err := jsonfilelog.New(info)
```

Comment:
We inherit all settings from [json-file](https://docs.docker.com/config/containers/logging/json-file/). Let's add them to our docs.